tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+##############################################################
+# Folders structure (28.07.2020)
+# NOTE: Drawing will be removed after approval script
+##############################################################
+# facelift/
+# ├─ build/
+# │  ├─
+# │  └─
+# ├─ ci/
+# │  ├─ appveyor
+# │  └─ travis
+# ├─ cmake/
+# ├─ CMakeLists.txt
+# ├─ src/
+# ├─ tests/
+# │  ├─ asyncfunctions/
+# │  ├─ benchmarking/
+# │  ├─ cmake/
+# │  ├─ CMakeLists.txt
+# │  ├─ combined/
+# │  ├─ coverage/
+# │  │   ├─ read.me
+# │  │   └─ test_gcov.sh*
+
+DIR_CVRG=$(cd "$(dirname "$0")"; pwd -P)
```

Comment:
revert to your previous commit. 
You do not need to use "$(cd "$(dirname "$0")"; pwd -P)" more than once.

Reply:
It is did just for traverse to top level.